test_runner/src/main/kotlin/ftl/cli/firebase/test/android/AndroidRunCommand.kt

```diff
@@ -42,10 +42,10 @@ class AndroidRunCommand : CommonRunCommand(), Runnable {
 
         val config = AndroidArgs.load(Paths.get(configPath), cli = this)
 
-        if (dumpShards) {
-            dumpShards(config)
-        } else runBlocking {
-            newTestRun(config)
+        runBlocking {
+            if (dumpShards)
+                dumpShards(config) else
+                newTestRun(config)
```

Comment:
In my opinion keeping curly braces is more redable 
```
runBlocking {
            if (dumpShards)
                dumpShards(config) else
                newTestRun(config)
      }
```

vs

```
runBlocking {
            if (dumpShards) {
                dumpShards(config)
            } else {
                newTestRun(config)
            }
        }
```
or
```
override fun run() {
        if (dryRun) {
            MockServer.start()
        }
        val config = AndroidArgs.load(Paths.get(configPath), cli = this)
        runProcess(config)
    }

    private fun runProcess(config: AndroidArgs) = runBlocking {
        if (dumpShards) {
            dumpShards(config)
        } else {
            newTestRun(config)
        }
    }
```
Tell me what you think.

Reply:
Curly braces make sense when u need to open new scope for more than one operation. Otherwise I prefer clear declarative expressions where curly braces are redundant.